#!/usr/bin/env python3

PACKAGE = "shifted_line_sim_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# param, data_type, 0, description, defaut, min, max
gen.add("white_low", int_t, 0, "threshold for detecting white", 211, 0, 255)
gen.add("lane_follow", bool_t, 0, "shifts the centroid's x position by -170 pixels", False)


# the third parameter must match this file's name
exit(gen.generate(PACKAGE, "line_follow", "LineFollow"))
